Formula/openjdk@11.rb

```diff
@@ -70,6 +70,7 @@ def install
       --with-debug-level=release
       --with-conf-name=release
       --with-jvm-variants=server
+      --with-jvm-features=shenandoahgc
```

Comment:
Why isn't this enabled by default? We're usually a bit reluctant to deviate from the default configuration since they're often chosen to be the most sensible defaults.

Reply:
> Why isn't this enabled by default? We're usually a bit reluctant to deviate from the default configuration since they're often chosen to be the most sensible defaults.

I think this is because it is a feature introduced since OpenJDK 12 and later back-ported to OpenJDK 11 (since 11.0.9). Actually many well-known JDK vendor enable this feature in their build (e.g., Zulu, Adoptium, RHEL, except for Oracle, see https://wiki.openjdk.java.net/display/shenandoah/Main#Main-JDKSupport). 

Enabling this feature does not change the behavior of existing applications but enable using `-XX:+UseShenandoahGC` if a user wants to.

Reply:
Is it enabled by default in OpenJDK 12 onwards?

Reply:
> Is it enabled by default in OpenJDK 12 onwards?

Yes.